h/app.py

```diff
@@ -139,8 +139,14 @@ def includeme(config):
             "h_pyramid_sentry.sqlalchemy_support": True,
         }
     )
-
     config.include("h_pyramid_sentry")
 
-    # pyramid-sanity should be activated as late as possible
-    config.include("pyramid_sanity")
+
+def _configure_mailer(config):
```

Comment:
This might be an arbitrary block to move to a private function. I could just add the `disable` and leave it at that?

Reply:
Maybe try and spot a few more? The CSP stuff looks like a likely candidate as well as maybe Jinja and retry/transaction stuff?

Reply:
Moved the CSP stuff which is a nice block. The others didn't look that good